### ISSUE TITLE
Error in NPC quote during quest (minor issue)

### ISSUE THREAD

**Issue by StarfallProjects** [dalaranwow/dalaran-wow#5040]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Valiance Keep - on the Quest The Hunt is On. On the Lower Docks, the NPC addressed me as $c, instead of by my class name (Paladin)
**Expected behaviour**: Tell us what should happen instead
Should use the class name

**Steps to reproduce the problem**:

1. Get the quest The Hunt is On
2. Go to the Lower Docks and use the orb
3. Talk to Tom Hegger
4. Tom says 'You don't know who you're messing with, $c!'

**Include proofs for this behaviour**
I assume this means proof of the correct behaviour?
His quotes are here: http://www.wowhead.com/npc=25827/tom-hegger

**Include Screenshots from the issue if necessary**
Screenshot attached
![minor-bug](https://user-images.githubusercontent.com/10284570/32627252-b3f59844-c589-11e7-8dd5-a9e5a8207452.png)

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/1376
